apps/web/src/server/applications/case/project-team/application-project-team.azure-service.js

```diff
@@ -77,6 +85,11 @@ export const getAllADUsers = async (ADToken) => {
  * @returns {Promise<ProjectTeamMember[]>}
  */
 const getAllCachedUsers = async (session) => {
+	if (config.authDisabled) {
+		// In development only, do not trigger any Azure request
+		return [];
+	}
+
```

Comment:
on local environments, this prevents requests to GRAPH API

Reply:
I'm wondering if we return a set of mock users when `authDisabled` - similarly to what happens with AD groups